gensrc/proto/segment_v2.proto

```diff
@@ -48,12 +48,12 @@ message MetadataPairPB {
 enum EncodingTypePB {
     UNKNOWN_ENCODING = 0;
     DEFAULT_ENCODING = 1;
-    PLAIN_ENCODING = 2; // for non-binary type
+    PLAIN_ENCODING = 2;
     PREFIX_ENCODING = 3;
     RLE = 4;
     DICT_ENCODING = 5;
     BIT_SHUFFLE = 6;
-    BINARY_PLAIN_ENCODING = 7;
```

Comment:
Is it OK to remove BINARY_PLAIN_ENCODING?

Reply:
Yes, we could reuse PLAIN_ENCODING for BinaryPlainPage, see https://github.com/apache/incubator-doris/pull/2050/files#diff-7c9224046521dc23a17299f3b1f4a849R60

Reply:
OK, I see